pkg/apis/cnsoperator/cnsvolumemetadata/v1alpha1/cnsvolumemetadata_types.go

```diff
@@ -71,6 +71,11 @@ type CnsVolumeMetadataSpec struct {
 	// namespace is set for objects whose EntityType is PERSISTENT_VOLUME.
 	//+optional
 	Namespace string `json:"namespace,omitempty"`
+
+	// K8sDistribution indicates the kubernetes distribution where the
+	// guest cluster is deployed
+	//+optional
+	ClusterDistribution string `json:"clusterdistribution,omitempty"`
```

Comment:
Shouldn't we update the CRD during CNS operator init?

Reply:
@SandeepPissay @RaunakShah I have tested the following scenarios.

1. When the SV is initially created with old image, the CRD is created with the old spec. Then I update the image for the SV CSI deployment to the new image, the CSI controller Pod is recreated and the CRD is also recreated with the new spec. But I am not sure what I manually did is same as how SV upgrade works. 
2. When the SV is initially created with new image, the CRD is created with new spec has cluster distribution.

Reply:
@SandeepPissay This is just a spec definition. Apart from CnsRegisterVolume, we are not creating any other APIs with a pre-defined structure which is currently a problem.